front/src/WebRtc/VideoPeer.ts

```diff
@@ -46,7 +51,68 @@ export class VideoPeer extends Peer {
         });
 
         this.userId = user.userId;
-        this.userName = user.name || '';
+        this.uniqueId = 'video_'+this.userId;
+
+        this.streamStore = readable<MediaStream|null>(null, (set) => {
```

Comment:
maybe this should be called a store since it is linked to another object. Maybe streamObservable? Samething for constraintsStore and statusStore